frontend-service/src/test/java/com/amazonaws/blox/frontend/integration/IntegrationTestConfig.java

```diff
@@ -48,6 +49,16 @@
     return handler;
   }
 
+  @Bean
+  public SampleController sampleController() {
+    return new SampleController();
+  }
+
+  @Bean
+  public Checker sampleControllerChecker() {
+    return mock(Checker.class);
```

Comment:
Did you figure out if this actually resets mocks after every test?

Reply:
Nope, will still need a reset for the mock once we have more test cases.  Will update to address this issue.